hw_19042018/customer.rb

```diff
@@ -0,0 +1,27 @@
+load 'person.rb'
+class Customer < Person
+  attr_accessor :indexID , :joindate 
+  def input
+    puts "enter name: "
```

Comment:
Use single quote for normal string.
Use double quote if insert variable to string.